src/java.desktop/macosx/classes/com/apple/laf/resources/aqua_zh_CN.properties

```diff
@@ -175,7 +175,7 @@ ProgressMonitor.progress.textAndMnemonic=\u8FDB\u5EA6...
 SplitPane.leftButton.textAndMnemonic=\u5DE6\u952E
 SplitPane.rightButton.textAndMnemonic=\u53F3\u952E
 # Used for Isindex
-IsindexView.prompt=\u8FD9\u662F\u53EF\u641C\u7D22\u7D22\u5F15\u3002\u8BF7\u8F93\u5165\u641C\u7D22\u5173\u952E\u5B57: 
+IsindexView.prompt=\u8FD9\u662F\u53EF\u641C\u7D22\u7D22\u5F15\u3002\u8BF7\u8F93\u5165\u641C\u7D22\u5173\u952E\u5B57:\u0020
```

Comment:
Inserted into a label. No use for a space afterwards. Other locales don't have it, but zh-TW does. Down further, there are identical-looking usages of `IsindexView.prompt`.